view/frontend/templates/cart/item/renderer/actions/product-protection-simple-offer.phtml

```diff
@@ -13,23 +14,29 @@ $productType = $product->getTypeId();
 $categoryName = '';
 
 if ($category = $product->getCategory()) {
-    $categoryName = $category->getName();
+  $categoryName = $category->getName();
 }
+
+$viewModel = $block->getData('viewModel');
+
+$shouldRender = $viewModel->isExtendProductProtectionEnabled() && $viewModel->isProductProtectionCartOfferEnabled();
```

Comment:
implemented `$shouldRender` logic based on both the primary PP toggle and the relevant display toggle setting. if either one is off, then the script will not be rendered. this is required because we are defaulting all offer display toggle settings to `true`, even though we are defaulting the main PP toggle to false.